tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/textual_stats_group_block.html

```diff
@@ -0,0 +1,11 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<div class="grid__stats-group stats-group">
+    {% if value.title %}
+        {% include "patterns/atoms/motif-heading/motif-heading.html" with heading_level=2 heading=value.title classes="motif-heading motif-heading--two-c motif-heading--static stats-group__heading" %}
```

Comment:
I think we've removed the motif-heading.html file, so we may need to update this.

Reply:
Yes, good catch 👍🏻